_posts/2023-10-18-efficient-filters-in-knn.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title:  Efficient Filtering in OpenSearch Vector Engine 
```

Comment:
Please update the title to sentence case: 

Efficient filtering in OpenSearch vector engine

Reply:
sure.